docs/package.json

```diff
@@ -22,9 +24,44 @@
     "develop": "gatsby develop",
     "fmt": "prettier --write \"**/*.{js,json,jsx,md,ts,tsx}\"",
     "fmt-check": "prettier --list-different \"**/*.{js,json,jsx,md,ts,tsx}\"",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
   },
   "devDependencies": {
-    "prettier": "^1.14.2"
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.6.0",
+    "enzyme": "^3.7.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^23.6.0",
+    "prettier": "^1.14.2",
+    "react-test-renderer": "^16.5.2"
```

Comment:
I think we should just move everything to `dependencies`. It doesn't have to block this PR. But there's no real need to separate them. It's often not obvious where a dep should go anyway, so all in `dependencies` is fine.

Reply:
I don't really care but you could make a case for things that require the build to go in deps while things like test, prettier, go into devdeps

Reply:
I'm not fussed either but adding them to `dependencies` will increase the `build-docs` step which is a pre-requisite for the deploy.

Reply:
Am I missing something, because in circle, `npm install` is run which will install all deps from package.json. Also, I would see that the way to do it is to:

- npm install
- cache node_modules
- restore cached node_modules and test
- restore cached node_modules and build

I've just found it not obvious where things go so default to just `dependencies`. Makes me have to make less decisions.

Reply:
Sorry you're right, I assumed we had `--production` flag for installing. This is going for awhile. I'll move them into `dependencies`